### DISCUSSION TITLE
OpenSSH SSH client

### DISCUSSION THREAD

**Discussion by eugenov** [Eugeny/tabby#3223] [General]
Just want to tell @Eugeny big TY for implementation of OpenSSH SSH client (that one inside powershell console). There is nothing close to such feature in other applications! It is something I did search for ages.

**Comment by Eugeny** [comment]
Appreciate the praise, but the `ssh` command in PowerShell is provided by Windows itself :) https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_overview

**Comment by eugenov** [comment]
Yeah, I know that, but i ment all the features inside one shell: full mouse support on remote hosts, X11, local ~\.ssh\config reading, and all that with terminus plugins. I believe the cause is fully integrated cygwin - in other projects its shell comes standalone...
